build/root/usr/local/bin/run

```diff
@@ -69,25 +70,26 @@ prechecks() {
 }
 
 entitle() {
-    # TODO: this should become an ansible playbook properly waiting
-    # for the deployment of the entitlement
-
-    if [[ "${SKIP_ENTITLEMENT:-n}" == y ]]; then
-        echo "Skipping entitlement (SKIP_ENTITLEMENT=${SKIP_ENTITLEMENT})"
+    if toolbox/entitlement/test.sh; then
+        echo "Cluster already entitled, skipping entitlement."
         return
     fi
 
+    if [[ "${INSIDE_CI_IMAGE:-n}" != y ]]; then
```

Comment:
I am a  bit lost, why using `INSIDE_CI_IMAGE` as the way to check this bit

Reply:
so the idea of the function
```
entitle() {
```

is that we do nothing if the cluster is already entitled (case 1: inside `toolbox/local-ci/deploy.sh`, case 2 : in the CI when we test `master` and `operatorhub` deployments in a row )
```
    if toolbox/entitlement/test.sh; then
        echo "Cluster already entitled, skipping entitlement."
        return
    fi
```
if `run` is executed outside of the CI image (shouldn't actually happened), there's nothing more we can do
```
    if [[ "${INSIDE_CI_IMAGE:-n}" != y ]]; then
        echo "FATAL: Cluster isn't entitled, cannot continue."
        exit 1
    fi
```
if `run` is executed inside the CI image, our entitlement file should be available
```
    PSAP_ENTITLEMENT="/var/run/psap-entitlement-secret/01-cluster-wide-machineconfigs.yaml"
    if [ ! -f "$PSAP_ENTITLEMENT" ]; then
        echo "FATAL: PSAP entitlement resources not found ($PSAP_ENTITLEMENT)"
        exit 1
    fi

    toolbox/entitlement/deploy.sh --machine-configs $PSAP_ENTITLEMENT
    if ! toolbox/entitlement/wait.sh; then
        echo "FATAL: Failed to properly entitle the cluster, cannot continue."
        exit 1
    fi
}
```

but let me change it a little bit

Reply:
I get it, but the `INSIDE_CI_IMAGE` check and then failing is a bit miss leading for an external reader trying to contribute to this repo. 
without much context a person can think it will fail inside a ci image 
maybe 
```bash
 if [[ "${INSIDE_CI_IMAGE:-n}" != y ]]; then
        echo "FATAL: This is intended to be use inside a CI cluster......"
        exit 1
fi
```
or something like that, so the user will realize he/she needs to entitle the cluster to run the playbooks manually